examples/helloworld/helloworld.py

```diff
@@ -0,0 +1,50 @@
+#   Copyright 2017 Covata Limited or its affiliates
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from covata.delta import Client, ApiClient, FileSystemKeyStore
+
+
+def main():
+    """
+    This example demonstrates the basics of creating identities, storing and
+    sharing secrets.
+
+    You will need to have a folder called "keystore" in your home directory or
+    will be created as a result of running this example.
+    """
+    key_store = FileSystemKeyStore("~/keystore/", b"passPhrase")
```

Comment:
I raised #67 but it might be a non-issue (we just enforce input into the constructor is byte type). If so we can close #67.